### DISCUSSION TITLE
Multiple Lorentz oscillator

### DISCUSSION THREAD

**Discussion by rugfri** [PyEllips/pyElli#165] [Q&A] [answered]
PyElli's, 
the Lorentz model encodes the possibility to superimpose multiple oscillators, each defined by one amplitude, energy, and broadening.
I have tried to introduce "repeated" parameters without success. Could you please share an example about how to describe 2 oscillators?
Thus with 2 amplitudes, energy and broadening parameters?

Thank you!

**Comment by domna** [comment, marked as answer]
Something like this should work:

```python
from elli.dispersions import LorentzEnergy

osc = LorentzEnergy().add(A=1, E=1, gamma=0.5).add(A=2, E=2, gamma=1)
```

which gives
```python
>>> print(osc)
LorentzEnergy
=============


Oscillators
===========
A = 1, E = 1, gamma = 0.5
A = 2, E = 2, gamma = 1
```
